.github/workflows/auto_release.yaml

```diff
@@ -0,0 +1,71 @@
+name: auto_release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
Almost every program allows to read the current version with something like `--version`. Since we would change that on every release is that something viable to listen to and trigger on that?

Reply:
My idea about our new self-update system and getting version via `--version` is:

* Add `RELEASE` file and commit it to master beforehand (just like [DXVK](https://github.com/doitsujin/dxvk/raw/master/RELEASE))
    * I think it's a good idea to set version tags to the "change RELEASE file" commits
* Get latest release URL from `https://github.com/lhark/truckersmp-cli/raw/master/RELEASE`
* Compare the version in `RELEASE` and local file `$XDG_DATA_HOME/truckersmp-cli/RELEASE`
* If the version in local file is older than master or the local file is not present, download `https://github.com/lhark/truckersmp-cli/releases/download/[version]/truckersmp-cli-[version].tar.xz` and unpack it into `$XDG_DATA_HOME/truckersmp-cli/`
* `--version` returns the version in `$XDG_DATA_HOME/truckersmp-cli/RELEASE`
* If update was done recently (within 1 hour? 1 day?), skip checking `RELEASE` and use local version

Reply:
Seems like good way. We have to keep in mind that a downloaded version can lay outside of `$XDG_DATA_HOME`.

But I was more into the trigger for this itself. Can we listen to the push which updates the `RELEASE` file and generate the tag and release these notes based on that? That way we wouldn't have to push the tag in an extra step.

Reply:
Yes, I think this is a great idea. Now a new tag will be set/pushed in GitHub Actions jobs when `RELEASE` file is changed.